.github/workflows/lint.yaml

```diff
@@ -15,14 +15,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
     - name: Select Rust toolchain with Clippy
       uses: actions-rs/toolchain@v1
```

Comment:
`actions-rs/toolchain@v1` also uses `node12` but [the repository for it](https://github.com/actions-rs/toolchain) is archived.